tests/packagedcode/data/conda/conda-yaml/test/environment_host_port.yml

```diff
@@ -0,0 +1,3 @@
+name: test
+dependencies:
+  - https://104.17.92.24:443::flask=1.0.2
```

Comment:
Is the URL a namespace, or is is rather a `repository_url` qualifier?

Reply:
It is a repository URL qualifier allright, there are common ones like conda-forge, anaconda, bioconda and the URL is usually not specified fully, but denoted by just a reference word.

Reply:
Actually I am not sure, this probably requires a bit more research, there are examples which seem more like a namespace, less like a repository URL:
* https://anaconda.org/r/r-terra `conda install r::r-terra`
* https://anaconda.org/lightsource2-dev/matplotlib `conda install lightsource2-dev::matplotlib`
Opening a follow up issue instead.

Reply:
#4094